doc/classes/Animation.xml

```diff
@@ -533,7 +533,7 @@
 			<argument index="1" name="interpolation" type="int" enum="Animation.InterpolationType">
 			</argument>
 			<description>
-				Set the interpolation type of a given track, from the INTERPOLATION_* enum.
+				Sets the interpolation type of a given track, from the[code]INTERPOLATION_*[/code] enum.
```

Comment:
Same as above. Also sorry for the comment spam, I guess I'll just start a review if I find anything else...